swoloszynski/journey-planner
# Add nodemon to restart server on app code changes, closes #19

## What
- Adds a package called [Nodemon](https://github.com/remy/nodemon)
- Nodemon watches for changes to app files and restarts the server when there are changes
- Added configs in package.json to add a delay and exclude test and doc files
- Changed `npm start` command to use nodemon

## How to test
- Stop server and restart with `npm start`
- Make a change to app.js and confirm that the server restarts (wait a second for the delay)
- Make a change to a test or doc file and confirm nothing happens
